src/main/java/net/pms/dlna/DLNAMediaDatabase.java

```diff
@@ -334,6 +335,27 @@ public synchronized void init(boolean force) {
 				sb.append(", TVEPISODENAME           VARCHAR2(").append(SIZE_MAX).append(')');
 				sb.append(", ISTVEPISODE             BOOLEAN");
 				sb.append(", EXTRAINFORMATION        VARCHAR2(").append(SIZE_MAX).append("))");
```

Comment:
@SubJunk one problem could be here.

Reply:
Nice catch, that was dumb of me :) I will refactor it to be harder to make that mistake